src/mon/MonCommands.h

```diff
@@ -441,6 +441,27 @@ COMMAND("mon remove " \
 COMMAND("mon rm " \
 	"name=name,type=CephString", \
 	"remove monitor named <name>", "mon", "rw", "cli,rest")
+COMMAND("mon features list " \
+        "name=feature_type,type=CephChoices,strings=all|monmap,req=false " \
+        "name=with_value,type=CephChoices,strings=--with-value,req=false", \
+        "list available mon map features to be set/unset", \
+        "mon", "r", "cli")
+COMMAND("mon features set " \
+        "name=feature_name,type=CephString " \
+        "name=feature_type,type=CephChoices,strings=persistent|optional,req=false " \
```

Comment:
in a way, I really dislike the idea of having users modifying the monmap features manually, so this was kind of a sledge hammer forcing the user to provide as much info as possible, even tho 'persistent' and 'optional' are not required (defaults to optional).

In any case, I'm fine with adding a 'set-persistent' instead. If we don't like it, in the future, we can always refactor this bit - but I'm guessing it will be fine, as these should not be used often.